File: P146PROD.cpp
#include<iostream>
using namespace std;

int main(void)
{
	int n, T1, T2, T3, count = 0;
	cin>>n;
	for(int i=0;i<n;i++)
	{
		cin>>T1>>T2>>T3;
		int num = 0;
		if(T1==1)
			num++;
		if(T2==1)
			num++;
		if(T3==1)
			num++;
		if(num>=2)
			count++;
	}
	cout<<count;
 return 0;
}

File: spoj py/P146PROD.py
from sys import exit
n = int(input())
count = 0
for i in range(0,n):
    num = 0
    T = raw_input()
    for i in T:
        if i=='1':
            num += 1
    if num>=2:
        count += 1
print count

exit(0)
